gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpRequestFormatter.java

```diff
@@ -30,22 +30,23 @@
 package com.google.api.gax.httpjson;
 
 import com.google.api.core.BetaApi;
-import com.google.gson.Gson;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
 
-/** Interface for classes that create parts of Http requests from a parameterized message. */
+/** Interface for classes that create parts of HTTP requests from a parameterized message. */
 @BetaApi
-public interface HttpRequestFormatter<MessageFormatT> {
+public interface HttpRequestFormatter<RequestMessageT> {
```

Comment:
Why rename the generic parameter from `MessageFormatT` when `HttpResponseFormatter` uses`MessageFormatT`? (it seems like they should be consistent)

Reply:
Ok, made them the same!